Volatile doesn't build on the latest nightly

```bash
error[E0599]: no method named `assert_len` found for type parameter `impl RangeBounds<usize>` in the current scope
   --> /home/kernel/.cargo/registry/src/github.com-1ecc6299db9ec823/volatile-0.4.3/src/lib.rs:627:17
    |
627 |         } = src.assert_len(self.reference.len());
    |                 ^^^^^^^^^^ method not found in `impl RangeBounds<usize>`

error: aborting due to previous error
```

```bash
$ rustc --version
rustc 1.52.0-nightly (476acbf1e 2021-03-03)
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -8,7 +8,7 @@
 #![no_std]
 #![cfg_attr(feature = "unstable", feature(core_intrinsics))]
 #![cfg_attr(feature = "unstable", feature(const_generics))]
-#![cfg_attr(feature = "unstable", feature(range_bounds_assert_len))]
+#![cfg_attr(feature = "unstable", feature(slice_range))]
 #![cfg_attr(feature = "unstable", allow(incomplete_features))]
 #![warn(missing_docs)]
 
@@ -25,6 +25,7 @@ use core::{
 use core::{
     intrinsics,
     ops::{Range, RangeBounds},
+    slice::range,
 };
 
 /// Allows creating read-only and write-only `Volatile` values.
@@ -624,7 +625,7 @@ where
         let Range {
             start: src_start,
             end: src_end,
-        } = src.assert_len(self.reference.len());
+        } = range(src, ..self.reference.len());
         let count = src_end - src_start;
         assert!(
             dest <= self.reference.len() - count,

